solutions/classes/car.js

```diff
@@ -0,0 +1,31 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, "utf-8");
+let idx = 0;
+data = data.split("\n");
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+// Define the required class here...
+const Car = class {
+  constructor(name = "Audi", model = "A4") {
+    this.name = name;
+    this.model = model;
+  }
+};
+
+// DO NOT CHANGE ANYTHING BELOW THIS LINE
+let flag = parseInt(readLine());
+if (flag === 1) {
+  var inputString = readLine().split(" ");
```

Comment:
@rpapavan97 
1. Please use `let` instead of `var`
2. Also, in the template, there is extra indentation for the `if .. `else` block

Reply:
modified the template and also added explanation to problem